ext/dom/tests/bug76738.phpt

```diff
@@ -5,7 +5,93 @@ dom
0PkFVU0JJTERVTkc8L2g0Pgo8dWw+CjxsaT4xOTg0IC0gMTk4OCBWb2xrc3NjaHVsZTwvbGk+CjxsaT4xOTg4IC0gMTk5MiBIYXVwdHNjaHVsZTwvbGk+CjxsaT4xOTkyIC0gMTk5MyBQb2x5dGVjaG5pc2NoZXIgTGVocmdhbmcgKE1pdCBBdXN6ZWljaG51bmcpPC9saT4KPGxpPjE5OTMgLSAxOTk3IExhbmRlc2JlcnVmc3NjaHVsZSAoTWl0IEF1c3plaWNobnVuZyk8L2xpPgo8L3VsPgo8aHI+CjxoND5JTlRFUkVTU0VOPC9oND4KPHVsPgo8bGk+TXVzaWssIEtpbm8sIDxhIGhyZWY9Imh0dHA6Ly93d3cua2FyYW9rZS13aWVuLmF0LyIgdGFyZ2V0PSJfYmxhbmsiIHRpdGxlPSJCYWJ1ZGVyJnJzcXVvO3MgfCBNYWNoIGRpZSBXZWx0IHp1IGRlaW5lciBCJnV1bWw7aG5lIHwgS2FyYW9rZSBpbiBXaWVuOiI+S2FyYW9rZTwvYT4KPC9saT4KPGxpPlNwb3J0IHNvZmVybmUgZXMgZGllIEZyZWl6ZWl0IHVuZCBkYXMgV2V0dGVyIHp1bGFzc2VuPC9saT4KPGxpPkVEViB1bmQgSW5mb3JtYXRpb25zdGVjaG5vbG9naWUgYXVjaCBwcml2YXQ8L2xpPgo8bGk+VW50ZXJoYWx0dW5nZWxla3Ryb25payBqZWdsaWNoZXIgQXJ0PC9saT4KPGxpPlNvZnR3YXJlLUVudHdpY2tsdW5nIG1pdCBWaXN1YWwgQmFzaWMgNi4wIHp1ciBwcml2YXRlbiBWZXJ3ZW5kdW5nPC9saT4KPGxpPlRlc3RlbiB2b24gU3lzdGVtZW4gdW5kIE5ldHplcmtlbiBpbSBwcml2YXRlbiB1bmQgZ2VzY2gmYXVtbDtmdGxpY2hlbiBVbWZlbGQ8L2xpPgo8bGk+SW50ZXJuZXQtUHJvZ3JhbW1pZXJ1bmcgKEphdmFTY3JpcHQgLyBDU1MgLyBQSFAgLyBNeVNRTCk8L2xpPgo8bGk+QXVkaW8tIHVuZCBCaWxkYmVhcmJlaXR1bmc8L2xpPgo8L3VsPgo8aHI+CjxoND5NVVNJSzwvaDQ+Cjx1bD4KPGxpPkhlYXZ5LU1ldGFsLCBSb2NrLCBLdXNjaGVsLVJvY2ssIE9sZGllcywgQXVzdHJvLVBvcDwvbGk+CjxsaT5EZWVwIFB1cnBsZSwgSm9lIEx5bm4gVHVybmVyLCBJYW4gR2lsbGFuLCBEZWYgTGVwcGFyZCwgQWVyb3NtaXRoLCBBQy9EQywgRG9ybywgQiZvdW1sO2hzZSBPbmtlbHosIE1ldGFsbGljYSwgR3VucyBOJmFjdXRlOyBSb3NlcywgSnVkYXMgUHJpZXN0LCBIZWxsb3dlZW4sIEtJU1MsIEFsaWNlIENvb3BlciwgQmxhY2sgU2FiYmF0aCwgTWFub3dhciwgTWFnbnVtLCBTYXZhdGFnZSwgVmljdG9yeSwgTGVkIFplcHBlbGluLCBHbGVubiBIdWdoZXMsIE51IFBhZ2FkaTwvbGk+CjxsaT5Kb2FuYSBaaW1tZXIsIFBldGVyIENldGVyYSwgQnJ5YW4gQWRhbXMsIFRvdG8sIFF1ZWVuLCBSRU0sIEV1cm9wZSwgU2NvcnBpb25zLCBXaGl0ZSBMaW9uLCBNZWF0IExvYWYsIEJvbiBKb3ZpLCBEaXJlIFN0cmFpdHMsIFJveGV0dGUsIENoaWNhZ28sIFNhbnRhbmEsIFN0YXR1cyBRdW8sIFN1cnZpdm9yLCBGb3JlaWduZXIsIEJvc3RvbiwgQm9uZmlyZSwgUmFpbmJvdywgR2VuZXNpcywgUG9pc29uLCBKb2huIE5vcnVtLCBSYWVtb25uPC9saT4KPGxpPlNUUywgSGVyYmVydCBHciZvdW1sO25lbXllciwgT3B1cywgRmFsY28sIFRvdGVuIEhvc2VuLCBXb2xmZ2FuZyBBbWJyb3MsIEEzLCBQZXRlciBNYWZmYXksIEtsYXVzIExhZ2UsIFB1ciwgUGV0ZXIgQ29ybmVsaXVzLCBIYW5zaSBEdWptaWM8L2xpPgo8L3VsPgo8aHI+CjxoND5TQ0hBVVNQSUVMRVIgLyBGSUxNRSAvIFNFUklFTjwvaDQ+Cjx1bD4KPGxpPkplYW4gQ2xhdWRlIFZhbiBEYW1tZSwgU3lsdmVzdGVyIFN0YWxsb25lLCBTdGV2ZW4gU2VhZ2FsLCBMb3JlbnpvIExhbWFzLCBOaWNvbGFzIENhZ2UsIFJpY2hhcmQgRGVhbiBBbmRlcnNvbiwgRGVuemVsIFdhc2hpbmd0b24sIENocmlzdG9waGVyIExhbWJlcnQsIE1lZyBSeWFuLCBTYXJhaCBNaWNoZWxsZSBHZWxsYXIsIEFtYW5kYSBUYXBwaW5nIC4uLi48L2xpPgo8bGk+SyZvdW1sO25pZ3JlaWNoIGRlciBIaW1tZWwsIEhlcnIgZGVyIFJpbmdlLCBIaWdobGFuZGVyLCBDb24gQWlyLCBUaGUgUm9jaywgU3RhcmdhdGUsIEluZGVwZW5kZW5jZSBEYXksIFp1bSB0Jm91bWw7dGVuIGZyZWlnZWdlYmVuLCBIYXJkIFRvIEtpbGwsIFRoZSBQYXRyaW90LCBSYW1ibywgUm9ja3ksIEhhcnRlIFppZWxlLCBUaW1lY29wLCBCZXN0IE9mIFRoZSBCZXN0LCBCcnVjZSBMZWUgU3RvcnkgLi4uLjwvbGk+CjxsaT5TdGFyZ2F0ZSwgQW5kcm9tZWRhLCBIaWdod2F5IFRvIEhlbGwsIFJlbmVnYWRlLCBOaWtpdGEsIFByb2ZpbGVyLCBDU0ksIERhcmsgQW5nZWwsIEJ1ZmZ5LCBDaGFybWVkPC9saT4KPC91bD4KPC9kaXY+CjwvZGl2Pgo=');
```

Comment:
I went ahead and delete this test entirely in https://github.com/php/php-src/commit/decf906a90c3e7b1cca2f226500c3ddc8fefd85e. It's a terrible test.